### PR TITLE
Sage docs odd scrolling issue

### PR DESCRIPTION
**Pull request by pixelflips**
## Description
Kept coming across odd scrolling behavior on the Sage Docs site where on the Objects and Elements pages, the page will continue to scroll for quite a ways after reaching the end of the content.

This is most likely a short-term fix since ideally the root cause should be addressed with a larger layout debug/refactor when time and manpower are available.

### Screenshots

|  before  |  after  |
|--------|--------|
|Screen Shot 2020-12-15 at 4 51 49 PM|Screen Shot 2020-12-15 at 4 53 56 PM|

### Steps for testing
1. Visit the elements or objects page and verify the page no longer scroll past the content.

## Related
-  N/A

### CODE REVIEW

**Review by monkeypox8 — APPROVED** [review]
👍🏼 😎 LGTM, thanks for fixing this annoying bug!
As we discussed, I'd expect we'll eventually clean up or overhaul the overall layout of the docs. There are some glaring, low-priority issues like the mobile nav being broken by all the buttons in the header.

**Review by QuintonJason — APPROVED** [review]

**Comment by philschanely** [comment]
Wow I never encountered this issue before. Yet this deploy did not seem to fix the issue for me. Did anyone else still see the issue in the preview deploy?

**Comment by QuintonJason** [comment]
> Wow I never encountered this issue before. Yet this deploy did not seem to fix the issue for me. Did anyone else still see the issue in the preview deploy?

Yeah, mine was good locally

File: packages/sage-assets/lib/stylesheets/layout/_page.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@ $-banner-height-offset: map-get($sage-banner-heights, default);
 .sage-page {
   display: flex;
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/layout/_page.scss:11]: @pixelflips I'm still seeing the issue in the deploy preview. How do things look for you if we remove the flex stuff and height here (keep the `overflow: hidden`) and then in `_sage-stage.scss` have this:

```scss
.sage-stage {
  display: flex;
  flex: 1;
  overflow: auto;
  height: calc(100vh - #{$sage-assistant-height});
}
```

Comment (pixelflips) [reply packages/sage-assets/lib/stylesheets/layout/_page.scss:11]: @philschanely - strange as it seems to work fine locally, but also not seeing it in the review app. For some reason, the new styles are not showing. Might be something related to the issue I added during migration where a similar thing was happening for me.

In either case, I will try the proposed adjustments above as soon as my morning meetings are done. Thanks for having an additional look!

**Comment by monkeypox8** [comment]
> > Wow I never encountered this issue before. Yet this deploy did not seem to fix the issue for me. Did anyone else still see the issue in the preview deploy?
> 
> Yeah, mine was good locally

Same here, local version was working but I haven't been able to consistently see style updates in deploy previews for quite some time.

**Comment by philschanely** [comment]
> > > Wow I never encountered this issue before. Yet this deploy did not seem to fix the issue for me. Did anyone else still see the issue in the preview deploy?
> > 
> > 
> > Yeah, mine was good locally
> 
> Same here, local version was working but I haven't been able to consistently see style updates in deploy previews for quite some time.

ok I've asked Smitty about this. My bad.

File: packages/sage-assets/lib/stylesheets/layout/_page.scss (hunk 1 above)
Comment (pixelflips) [reply packages/sage-assets/lib/stylesheets/layout/_page.scss]: @philschanely that works great as well. Thanks again for the suggestion!

**Review by philschanely — APPROVED** [review]

**Comment by philschanely** [comment]
Before merging, @pixelflips please rebase and set to merge to `develop`.

**Pull request merged**